src/pages/pse-sponsorships/apply.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Stack } from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from 'next/image';
+
+import { PageMetadata, PageSubheading } from '../../components/UI';
+
+import pseSponsorshipsHero from '../../../public/images/pse-sponsorships.png';
+
+const PSESponsorshipsApply: NextPage = () => {
+  return (
+    <>
+      <PageMetadata
+        title='PSE Sponsorships Application'
+        description='Submit Sponsorship Application to Privacy & Scaling Explorations.'
+      />
+      <Head>
+        <meta name='robots' content='noindex' />
+      </Head>
+
+      <Box
+        position='relative'
+        py={{ md: 12 }}
+        px={{ sm: 5, md: 24, lg: 32, xl: 72 }}
+        pb={5}
+        mt={{ base: 24, md: 6 }}
+      >
+        <Image
```

Comment:
The image is not centered.

![image](https://github.com/ethereum/esp-website/assets/468158/72e050fc-7875-47d1-a337-ed1d6170ac42)

I'd recommend wrapping it with `<Center>`.

Reply:
Fixed